### DISCUSSION TITLE
reply_message_to_id PROBLEM

### DISCUSSION THREAD

**Discussion by jdavidclaro** [telegraf/telegraf#1582] [Q&A] [answered]
HELLO, I'm having problems with reply_message_to_id. It is supposed to reply with the message send by the user and the bot message, bot only reply a normal message. What could be the reason? THANKS

**Comment by MKRhere** [comment]
Could you explain more, provide some code sample?

**Reply by jdavidclaro** [reply]
![Bot_Reply_2022-02-16 01-32-22](https://user-images.githubusercontent.com/73051277/154209521-75dc12e1-f1c5-484f-96da-70d948454e88.png)
I'm trying the bot reply to the messages this way.

![reply_to_message_id_TelegramAPI](https://user-images.githubusercontent.com/73051277/154209575-515a09a1-7ccd-497a-87d2-b71386acd4ab.png)
So I find this parameter in the method sendMessage

![code](https://user-images.githubusercontent.com/73051277/154210615-d859926a-d1f2-4786-a776-edcd43fe669d.png)
This is the code.

![Bot_2022-02-16 01-28-22](https://user-images.githubusercontent.com/73051277/154209646-b4b6a364-b3d5-4b82-93dc-5dcbedc21d8b.png)
But nothing happens, at lees no what I expect

**Reply by MKRhere** [reply]
Not sure what's happening here, but you don't need to use `reply_to_message_id`. You can use the `ctx.reply()` shortcut for this. In your sample code, it'd just be `ctx.reply("???")`.

**Reply by jdavidclaro** [reply]
Of course, the simplest way it's ctx.reply(). But I'm trying to make the bot answer reply to a specific message, like other bots I'm using do and Iook like I show you in the first picture. I'm trying it using repply_to_message_id that should do somthing, whatever it is, but nothing look different. So, maybe that it's a telegraf problem.

**Comment by hypernova7** [comment, marked as answer]
I think you're doing it wrong, it's reply_to_message_id not reply_message_to_id. Try doing it this way
```javascript
bot.on('message', ctx => {
  ctx.reply('Hi there', {
    reply_to_message_id: ctx.message.message_id
  })
})
```

**Reply by jdavidclaro** [reply]
YOU'RE ABSOLUTELY RIGHT. THANKS A LOT
